src/Entities/ExpOrb.h

```diff
@@ -18,8 +18,8 @@ class cExpOrb :
 
 	CLASS_PROTODEF(cExpOrb)
 
-	cExpOrb(double a_X, double a_Y, double a_Z, int a_Reward);
-	cExpOrb(const Vector3d & a_Pos, int a_Reward);
+	cExpOrb(double a_X, double a_Y, double a_Z, int a_Reward, float a_SpeedX = 0.f, float a_SpeedY = 0.f, float a_SpeedZ = 0.f);
+	cExpOrb(const Vector3d & a_Pos, int a_Reward, float a_SpeedX = 0.f, float a_SpeedY = 0.f, float a_SpeedZ = 0.f);
```

Comment:
Is this change really adding value? Why not just set the speed after construction, that's how it's done for TNT already:
https://github.com/cuberite/cuberite/blob/98d807187bbaec4cd65dbe7c1ecac0435efb6df0/src/World.cpp#L2337-L2353

Reply:
experience orbs need to created with speed if multiple orbs are generated.

There are similar operation in method **cWorld::SpawnItemPickups**

Reply:
I agree I can set speed after create entity, but I think these are just two different methods to implement same feature, is it neccessary to create new commit to modify it ?

Reply:
Well, the interface would need to be changed either way because it isn't using `Vector3d` (see #3853).

Reply:
code updated for set speed after orb created